staging/copy.sh

```diff
@@ -147,7 +148,8 @@ cp -r "${CLIENT_REPO_TEMP}"/vendor/k8s.io/kubernetes/* "${CLIENT_REPO_TEMP}"/
 rm -rf "${CLIENT_REPO_TEMP}"/vendor
 
 echo "rewriting Godeps.json"
-# The entries for k8s.io/apimahcinery are not removed from Godeps.json, though
+
```

Comment:
Could you keep this file unchanged? https://github.com/kubernetes/kubernetes/pull/44784 will get rid of most of this script.